### PR TITLE
[Dat 9857] Remove Derby ignore

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/resources/liquibase/harness/change/expectedSql/derby/setColumnRemarks.sql [hunk 1]
```diff
@@ -1,3 +1,4 @@
 INVALID TEST
--- Liquibase doesn't support this change type for Derby DB
--- https://docs.liquibase.com/change-types/set-column-remarks.html
\ No newline at end of file
+
+Liquibase doesn nott support this change type for Derby DB
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSql/derby/setColumnRemarks.sql:3]: typo

File: src/main/resources/liquibase/harness/change/expectedSql/derby/setTableRemarks.sql [hunk 2]
```diff
@@ -1,3 +1,4 @@
 INVALID TEST
--- Liquibase doesn't support this change type for Derby DB
--- https://docs.liquibase.com/change-types/set-table-remarks.html
\ No newline at end of file
+
+Liquibase doesn not support this change type for Derby DB
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSql/derby/setTableRemarks.sql:3]: typo

File: src/main/resources/liquibase/harness/change/expectedSql/derby/mergeColumns.sql [hunk 3]
```diff
@@ -1,3 +1,4 @@
 INVALID TEST
--- Liquibase doesn't support mergeColumns change type for Derby:
--- https://docs.liquibase.com/change-types/merge-columns.html
\ No newline at end of file
+
+Liquibase doesn not support mergeColumns change type for Derby:
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSql/derby/mergeColumns.sql:3]: same typo

File: src/main/resources/liquibase/harness/change/expectedSql/derby/dropFunction.sql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+INVALID TEST
+
+Drop function for Apache Derby is not supported by Liquibase:
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSql/derby/dropFunction.sql:3]: add link

File: src/main/resources/liquibase/harness/change/expectedSql/derby/addAutoIncrement.sql [hunk 5]
```diff
@@ -1,4 +1,4 @@
 INVALID TEST
 
-addAutoIncrement is not supported for Derby
---https://docs.liquibase.com/change-types/add-auto-increment.html
\ No newline at end of file
+Renaming trigger for Apache Derby is not supported by Liquibase:
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/expectedSql/derby/addAutoIncrement.sql:3]: wrong copy-paste

File: pom.xml [hunk 6]
```diff
@@ -153,7 +153,12 @@
         <dependency>
             <groupId>org.apache.derby</groupId>
             <artifactId>derbyclient</artifactId>
-            <version>10.15.2.0</version>
+            <version>10.14.2.0</version>
```
Comment (KushnirykOleh) [inline pom.xml:156]: can we keep newer version of both dependencies here?

File: .gitignore [hunk 7]
```diff
@@ -7,7 +7,7 @@ bin/
 build/
 .project
 .DS_Store
-src/test/resources/derby/lbcat/*
+
```
Comment (KushnirykOleh) [inline .gitignore:10]: i think we need to keep this ignored

File: pom.xml (hunk 6 above)
Comment (AlexanderSashchenko) [reply pom.xml:156]: No. Newer version requires Java9

File: src/main/resources/liquibase/harness/change/expectedSql/derby/addAutoIncrement.sql (hunk 5 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/change/expectedSql/derby/addAutoIncrement.sql:3]: fixed

File: src/main/resources/liquibase/harness/change/expectedSql/derby/setColumnRemarks.sql (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/change/expectedSql/derby/setColumnRemarks.sql:3]: fixed

**Review by KushnirykOleh — APPROVED** [review]

**Pull request merged**